channels/ON-CHAIN.md

```diff
@@ -166,6 +167,34 @@ on-chain transactions that have either `round` or a `payload` (with a `round` in
 
 (***TODO***: should a channel be considered closed if all the tokens are taken from it?)
 
+### `channel_snapshot_solo`
```

Comment:
Shall this be somewhere in overview https://github.com/aeternity/protocol/blame/01d13252319c6b2ab97356fe9208336243025a15/channels/README.md#L326 ?